config/core/300-resources/stagepodautoscaler.yaml

```diff
@@ -80,6 +80,10 @@ spec:
                   description: ActualScale shows the actual number of replicas for the revision.
                   type: integer
                   format: int32
+                replicasTerminating:
```

Comment:
I have to break the rule of not changing crd in the same release, coz this line is crucial.